### PR TITLE
Readability

### PR DESCRIPTION
**Pull request by KoryNunn**

### CODE REVIEW

**Comment by KoryNunn** [comment]
@DevBonBon used your readability branch, fixed up a few things and swapped to arrow functions to cut down file size.

Let me know if you see anything out of place.

File: package.json [hunk 1]
```diff
@@ -7,15 +7,17 @@
...
   "devDependencies": {
     "browserify": "^14.4.0",
     "tape": "^3.0.3",
-    "uglify-js": "^3.4.9"
+    "uglify-es": "^3.3.9"
   },
   "scripts": {
     "test": "npm run testBuild && google-chrome ./test/index.browser.js",
-    "testBuild": "browserify ./test/index.js > ./test/index.browser.js",
+    "testBuild": "browserify ./test/index.js > ./test/index.browse.js",
```
Comment (DevBonBon) [inline package.json]: missing 'r'

File: package.json [hunk 2]
```diff
@@ -7,15 +7,17 @@
   ],
   "version": "3.1.0",
   "main": "crel.js",
-  "dependencies": {},
+  "dependencies": {
+    "uglifyjs": "^2.4.11"
```
Comment (DevBonBon) [inline package.json]: Is this supposed to be here?

File: package.json (hunk 1 above)
Comment (DevBonBon) [reply package.json]: this is propably why there was a new file added

**Comment by DevBonBon** [comment]
I'm not sure what you mean by increased file size. The min file stayed exactly the same size, and that should be the only version used in production
You could create a sepperate crel.es version, with minified builds and such. Other packages are doing it, so why not crel? uglify3 is backward compatible afaik

**Comment by KoryNunn** [comment]
Did you run `npm run build`? The bundle was about 10% larger for me before swapping to arrows.

I'm not super keen on supporting two versions of crel (es and js), and there are existing version on cdnjs and npm if people need to target old browsers.

I'll merge this since it looks like it's good to go at this point.

**Comment by DevBonBon** [comment]
Ah, well I don't know about the bundle, I was only looking at the build results :P
Would you oppose it, if i were to make an es version and pr it?

**Comment by KoryNunn** [comment]
Honestly, I would. I like to keep things basic. The existing implementation exists in npm, if people want latest they can have modern js features.

**Comment by DevBonBon** [comment]
Fair enough, polyfill exists anyways if people really need the newer versions

**Pull request merged**